src/views/v2/widgets/events-list/event/date-tag.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Widget: Events List Event Venue
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/widgets/events-list/event/venue.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
+ *
+ * @see tribe_get_event() For the format of the event object.
+ */
+
+use Tribe__Date_Utils as Dates;
+
+/*
+ * If the request date is after the event start date, show the request date to avoid users from seeing dates "in the
+ * past" in relation to the date they requested (or today's date).
+ */
+$display_date = empty( $is_past ) && ! empty( $request_date )
+	? max( $event->dates->start_display, $request_date )
+	: $event->dates->start_display;
+
+$event_month     = $display_date->format_i18n( 'M' );
+$event_day_num   = $display_date->format_i18n( 'j' );
+$event_date_attr = $display_date->format( Dates::DBDATEFORMAT );
+?>
```

Comment:
```suggestion
?>

```

Reply:
@lucatume not sure why this needs an extra space?

Reply:
Formatting habit really.